components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/backchannellogout/LogoutRequestSender.java

```diff
@@ -118,7 +167,7 @@ public void sendLogoutRequests(String opbsCookieId, String tenantDomain) {
             for (Map.Entry<String, String> logoutTokenMap : logoutTokenList.entrySet()) {
                 String logoutToken = logoutTokenMap.getKey();
                 String bcLogoutUrl = logoutTokenMap.getValue();
-                LOG.debug("A logoutReqSenderTask will be assigned to the thread pool");
+                LOG.info("A LogoutReqSenderTask will be assigned to the thread pool.");
```

Comment:
Do we need to make this is Info? IMO debug should be good. Otherwise, in happy path, this will fill up the logs.

Reply:
Addressed via 696835c1e65a4a4e34981c2dfe9bc7c9ccd23e71